Podfile.lock

```diff
@@ -34,7 +34,7 @@ DEPENDENCIES:
   - Quick (~> 2.0)
 
 SPEC REPOS:
-  https://github.com/cocoapods/specs.git:
+  trunk:
```

Comment:
With CocoaPods 1.10.0, after running `pod install` this gets replaced by `https://cdn.cocoapods.org/:`.

Reply:
This seems to have been an issue with my repository configuration for CocoaPods. Nevermind.